### PR TITLE
standardized inference and number of iterations for mSuperb single lang track

### PR DESCRIPTION
**Pull request by DanBerrebbi**

### CODE REVIEW

File: egs2/msuperb/asr1/conf/tuning/train_asr_fbank_single.yaml [hunk 1]
```diff
@@ -30,14 +24,14 @@ num_workers: 4
 batch_type: sorted
 batch_size: 8
 accum_grad: 4
-max_epoch: 50
+max_epoch: 200
```
Comment (ftshijt) [inline egs2/msuperb/asr1/conf/tuning/train_asr_fbank_single.yaml:27]: I think we can just delete this line?

File: egs2/msuperb/asr1/conf/tuning/train_asr_s3prl_single.yaml [hunk 2]
```diff
@@ -43,14 +36,14 @@ num_workers: 4
 batch_type: sorted
 batch_size: 8
 accum_grad: 4
-max_epoch: 50
+max_epoch: 200
```
Comment (ftshijt) [inline egs2/msuperb/asr1/conf/tuning/train_asr_s3prl_single.yaml:39]: ditto

File: egs2/msuperb/asr1/run_single_lang.sh [hunk 3]
```diff
@@ -39,6 +39,7 @@ for duration in 10min 1h; do
             --feats_type raw \
             --asr_config "${asr_config}" \
             --inference_config "${inference_config}" \
+            --inference_asr_model "valid.loss.ave_5best.pth" \
```
Comment (ftshijt) [inline egs2/msuperb/asr1/run_single_lang.sh:42]: I think the default one is the ave ones right? In that case, no need to change it

Comment (DanBerrebbi) [reply egs2/msuperb/asr1/run_single_lang.sh:42]: oh yes I forgot about this softlink

**Comment by ftshijt** [comment]
I think you also need some updates to some other scripts (due to the version change)

**Comment by DanBerrebbi** [comment]
> 

Yes I saw it
I'll update after I make sure the current files work well without any bug which I am doing now

**Review by ftshijt — APPROVED** [review]
Looks good to me! thanks

**Pull request merged**